src/listconf.c

```diff
@@ -353,16 +356,14 @@ static void listconf_list_build_info(void)
 	printf("Terminal locale string: %s\n", john_terminal_locale);
 	printf("Parsed terminal locale: %s\n", cp_id2name(options.terminal_enc));
 
-#ifdef AESNI_IN_USE
-	if (using_aes_asm())
-		printf("AES-NI: available\n");
-	else
-		printf("AES-NI: not supported by CPU\n");
-#elif defined(__i386__) || defined(__x86_64__)
-	printf("AES-NI: not built\n");
-#else
-	printf("AES-NI: not applicable\n");
+	printf("AES hardware acceleration: %s\n",
```

Comment:
I think we should move the `AES hardware acceleration` line to be near the SIMD line in the beginning, not at the end of list.